src/app/containers/Index/Title/index.jsx

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable jsx-a11y/aria-role */
+import React, { useContext } from 'react';
+import { node, bool } from 'prop-types';
+import styled from 'styled-components';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
+import { getParagon, GEL_FF_REITH_SANS } from '@bbc/gel-foundations/typography';
+import {
+  GEL_GROUP_2_SCREEN_WIDTH_MIN,
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
+  GEL_GROUP_4_SCREEN_WIDTH_MIN,
+} from '@bbc/gel-foundations/breakpoints';
+import {
+  GEL_SPACING,
+  GEL_SPACING_DBL,
+  GEL_SPACING_TRPL,
+  GEL_SPACING_QUAD,
+} from '@bbc/gel-foundations/spacings';
+import { C_EBON } from '@bbc/psammead-styles/colours';
+import { ServiceContext } from '#contexts/ServiceContext';
+
+const Heading = styled.h1`
+  ${({ script }) => (script ? getParagon(script) : '')};
+  font-family: ${GEL_FF_REITH_SANS};
+  color: ${C_EBON};
+  margin: 0;
```

Comment:
Do we need this?

Reply:
This was to reset the H1 margin - I need actual designs for this title once it's past POC stage though!